.github/memory_statistics_config.json

```diff
@@ -1,9 +1,11 @@
 {
     "lib_name": "AWS IoT Jobs",
     "src": [
-        "source/jobs.c"
+        "source/jobs.c",
+        "coreJSON/source/core_json.c"
     ],
     "include": [
-        "source/include"
+        "source/include",
+        "coreJSON/source/include"
```

Comment:
Same as [previous comment](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/pull/85/files#r1358667369)